prboom2/src/gl_missingtexture.c

```diff
@@ -133,22 +133,22 @@ static void gld_PrepareSectorSpecialEffects(void)
 
           /* now mark the sectors that may require HOM bleed-through */
           if (needs_front_upper && side0->toptexture == NO_TEXTURE) {
-            side1->sector->flags |= MISSING_TOPTEXTURES;
+            side0->sector->flags |= MISSING_TOPTEXTURES;
```

Comment:
Why were the sides flipped before? Was it a bug?

Reply:
Yes, as far as I can tell.  I don't see how it could have worked before.